packages/cli/src/services/cache.service.ts

```diff
@@ -82,9 +84,12 @@ export class CacheService {
 	): Promise<unknown> {
 		const value = await this.cache?.store.get(key);
 		if (value !== undefined) {
+			this.emit(MetricsCounterEvents.cacheHit);
```

Comment:
same as with eventbus: the event emitter should own the names of the events it emits. that way there are no metrics related imports here, and `MetricsService` becomes an isolated unit.